recipe/patches/qvksc_dirty_fix.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/src/gui/rhi/qrhivulkan.cpp b/src/gui/rhi/qrhivulkan.cpp
+index 064c197af9..14fea10aad 100644
+--- a/src/gui/rhi/qrhivulkan.cpp
++++ b/src/gui/rhi/qrhivulkan.cpp
```

Comment:
oh the qtbase prefix should be added because I made the patch from the git repo:
```
--- a/qtbase/src/gui/rhi/qrhivulkan.cpp
+++ b/qtbase/src/gui/rhi/qrhivulkan.cpp
```